.github/workflows/openapi_check.yaml

```diff
@@ -1,14 +0,0 @@
-on: [push, pull_request]
```

Comment:
why delete the openapi check?

Reply:
For now I removed linting out of the deutschland-generator-action since I agree that it should be a prestep and the generation action is only executed on success of it. 

Probably we want to run generation only on a new tag and pull the version from the tag somehow.

The linting for me is constantly failing with this message:
No ruleset has been found. Please provide a ruleset using the --ruleset CLI argument, or make sure your ruleset file matches .?spectral.(js|ya?ml|json)

Not sure why though.